gradle-plugin/src/main/groovy/ch/silviowangler/gradle/restapi/Consts.java

```diff
@@ -25,7 +25,11 @@
 
 public class Consts {
 
-    public static final String CONFIGUATION_REST_API = "restApiSpecification";
+    public static final String CONFIGURATION_REST_API = "restApiSpecification";
 
     public static final String TASK_GROUP_REST_API = "REST API";
+
+    public static final String ISO_LOCAL_DATE_FORMAT = "yyyy-MM-dd";
+    public static final String ISO_LOCAL_TIME_FORMAT = "HH:mm:ss";
+    public static final String ISO_LOCAL_DATE_TIME_FORMAT = String.format("%sT%s", ISO_LOCAL_DATE_FORMAT, ISO_LOCAL_TIME_FORMAT);
```

Comment:
This does not seem to be a valid UTC datetime pattern. A UTC date/timestamp looks like 

- `2001-03-01T14:00:01Z`

Does this pattern covers all of this? Does it especially cover the timezone information?

Reply:
Good point, I tried this manually, I couldn't get this to work with `Instant`. Looks like Micronaut's `TimeConverterRegistrar` class doesn't support `CharSequence -> Instant`. 

It works fine however with `LocalDateTime` if the "T" in the `ISO_LOCAL_DATE_TIME_FORMAT` is omitted. 

I'm not sure how to solve this, maybe in the meantime I can remove the `@Format` annotation for `datetime` so that at least `date` will work?